saskatoon/saskatoon/urls.py

```diff
@@ -10,3 +10,8 @@
     path('',include('django.contrib.auth.urls')),
     path('__debug__/', include(debug_toolbar.urls)),
 ]
+
+if 'rosetta' in settings.INSTALLED_APPS:
+    urlpatterns += [
+        re_path(r'^rosetta/', include('rosetta.urls'))
+    ]
```

Comment:
Since rosetta is always enabled, I think it's safe to do like other `urlpatterns`, and include it directly in the list.

```suggestion
    path('rosetta/', include('rosetta.urls')),
]
```

Reply:
done